### ISSUE TITLE
Partition data set into: Training Set, Testing Set, and Validation Set for method 2: cross-validation technique 

### ISSUE THREAD

**Issue by calamari99** [calamari99/Facebook-Post-Predictor#3]
- not required but may be nice to include later on!
